api/api.go

```diff
@@ -80,77 +80,61 @@ const (
 // Node describes the state of a node.
 // It includes the current physical state (CPU, memory, storage, network usage) as
 // well as the containers running on the system.
+//
+// swagger:model
 type Node struct {
-	Id        string
-	Cpu       float64 // percentage.
-	MemTotal  uint64
-	MemUsed   uint64
-	MemFree   uint64
-	Avgload   int
-	Status    Status
+	// Id of the cluster.
+	Id string
+	// Id of the cluster.
+	Cpu float64 // percentage.
+	// Total Memory of the node
```

Comment:
Shouldnt this be of the form
MemTotal is the total memory of the node
The same applies to the rest of the fields as well.

Reply:
You are right i'll change that